src/pages/Analysis/index.js

```diff
@@ -0,0 +1,181 @@
+import styles from './index.less';
+import AnalysisChart from '@/components/AnalysisChart';
+import { donutChart1, donutChart2, barGraph1, barGraph2 } from '../../../mock/analysis';
+
+const legends = {
+  'pbench-user-benchmark': 'blue',
+  trafficgen: 'cyan',
+  uperf: 'orange',
+  linpack: 'gold',
+  fio: 'purple',
+};
+
+const legendField = Object.keys(legends).map(function(key) {
+  return (
+    <div>
+      <span className={styles.colorBox} style={{ backgroundColor: legends[key] }} />
+      <span className={styles.legendLabel}>{key}</span>
+    </div>
+  );
+});
+
+function Analysis() {
+  return (
+    <div className={styles.paddingBig}>
+      <TextContent className={styles.paddingSmall}>
+        <Grid>
+          <GridItem span={6}>
+            <Text component={TextVariants.h1}> Analysis</Text>
+          </GridItem>
+          <GridItem span={6}>
+            <Text component={TextVariants.h3} style={{ textAlign: 'right' }}>
+              {' '}
+              Total runs: 1,179
+            </Text>
+          </GridItem>
+        </Grid>
+      </TextContent>
+      <Grid className={styles.paddingSmall}>
+        <GridItem span={12}>
+          <Card className={styles.borderedCard}>
+            <div className={styles.paddingBig}>
+              <TextContent>
+                <Text component={TextVariants.h2}>
+                  {' '}
+                  Runs
+                  <Button variant="link" icon={<ExternalLinkAltIcon />} style={{ float: 'right' }}>
+                    View in Kibana
+                  </Button>
+                </Text>
+              </TextContent>
+            </div>
+          </Card>
+          <Card className={styles.borderedCard}>
+            <Grid>
+              <GridItem span={7}>
+                <TextContent className={styles.paddingMedium}>
+                  <Text component={TextVariants.h3} style={{ color: 'grey' }}>
+                    {' '}
+                    Click on sections of the graph to view detailed data
+                  </Text>
+                </TextContent>
+                <Grid>
+                  <GridItem span={5} offset={1}>
+                    <div style={{ height: '260px', width: '260px' }} className={styles.paddingBig}>
+                      <AnalysisChart chartType="donut" usedSpace={donutChart1} />
+                    </div>
+                  </GridItem>
+                  <GridItem span={6}>
+                    <div style={{ height: '260px', width: '260px' }} className={styles.paddingBig}>
+                      <AnalysisChart chartType="donut" usedSpace={donutChart2} />
+                    </div>
+                  </GridItem>
+                </Grid>
+              </GridItem>
+              <GridItem span={5} style={{ borderLeft: '1px solid #b3b2b2' }}>
+                <div className={styles.paddingBig}>
+                  <TextContent>
+                    <Text component={TextVariants.h3}>example.eng.bos.redhat.com</Text>
```

Comment:
Can the hostname here be part of your mock payload rather than hardcoded here? (And below...) Seems like this isn't really at "real component with mock data" since part of the component itself seems to be faked.